site/index0.html

```diff
@@ -3,15 +3,15 @@
 <head>
 <script src="https://code.jquery.com/jquery-latest.min.js" type="text/javascript"></script>
 <script src="https://cdnjs.cloudflare.com/ajax/libs/spin.js/2.3.2/spin.min.js" type="text/javascript"></script>
-<script type="text/javascript" src="/epidata/lib/delphi_epidata.min.js"></script>
+<script type="text/javascript" src="https://delphi.cmu.edu/epidata/lib/delphi_epidata.min.js"></script>
```

Comment:
Why don't relative paths work?

Reply:
I changed that cause during development you don't have it at this path. Once we publish the JS client to NPM we can use a similar CDN link like for jquery